src/vcpkg/portfileprovider.cpp

```diff
@@ -193,6 +194,7 @@ namespace vcpkg::PortFileProvider
                     }
                     else
                     {
+                        Metrics::g_metrics.lock()->track_property("versioning-error-version", "defined");
```

Comment:
This is a bit imprecise because we don't actually know whether this is a termination-worthy error, however it may be intractable to move this to every `load_control_file().value_or_exit()`.

Reply:
The more invasive but "correct" approach would be to create a new type like:
```cpp
struct TrackedError {
    std::string message;
    std::string metric;
};
```
which upon calling `Expected<T,TrackedError>::value_or_exit()` would track the property just in time to exit. However, I think that is probably too invasive at this time.

Reply:
I may attempt to track in `value_or_exit()` in a separate PR.
For the time being, I'll just merge with the current metrics if there are no objections.